Kiss.Bff.EndToEndTest/BaseTestInitializer.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Playwright;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+
+
+namespace PlaywrightTests
+{
+    [TestClass]
+    public class BaseTestInitializer
+    {
+        protected IPlaywright Playwright;
+        protected IBrowser Browser;
+        protected IBrowserContext Context;
+        protected IPage Page;
+        protected IPage FrontendPage;
+        protected AzureAdLoginHelper LoginHelper;
+        protected IConfiguration Configuration;
+
+        [TestInitialize]
+        public virtual async Task TestInitialize()
+        {
+            try
+            {
+                Configuration = new ConfigurationBuilder()
+                    .AddUserSecrets<BaseTestInitializer>()
+                    .AddEnvironmentVariables()      
+                    .Build();
+
+                var username = Configuration["TestSettings:TEST_USERNAME"];
```

Comment:
Als je een functie maakt om een config waarde op te halen, die meteen checkt of die waarde null of empty is, kan je denk ik relatief eenvoudig per config waarde een informatieve foutmelding tonen. Zoiets:
```csharp
private string GetRequiredConfig(IConfiguration configuration, string key)
{
  var value = configuration[key];
  if(string.IsNullOrEmpty(value))
  {
    throw new InvalidOperationException($"'{key}' is missing from the configuration");
  }
  return value;
}
```